src/Utilities/Compiler/Extensions/ReportDiagnosticExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static string ToAnalyzerConfigString(this ReportDiagnostic reportDiagnost
                 ReportDiagnostic.Warn => DiagnosticSeverity.Warning,
                 ReportDiagnostic.Info => DiagnosticSeverity.Info,
                 ReportDiagnostic.Hidden => DiagnosticSeverity.Hidden,
-                ReportDiagnostic.Suppress => null,
+                ReportDiagnostic.Suppress => default(DiagnosticSeverity?),
```

Comment:
I will revert this change. I don't know why but I have to do this change to have the solution compiling.

Reply:
I think you probably need to upgrade to a later VS2019 build. Target-typed switch expressions was not in the original implementation of C# 8 switch expressions, it was done as a bug fix, and likely your version of VS still does not have that fix.